graphql-schemas/schema.yml

```diff
@@ -3107,6 +3107,7 @@ confs:
   - { name: self_service, type: SelfServiceConfig_v1, isList: true }
   - { name: cloudflare_access, type: CloudflareAccountRole_v1, isList: true }
   - { name: ldapGroup, type: string }
+  - { name: memberSource, type: RoleMembershipSource_V1 }
```

Comment:
should that be a list?

Reply:
i kept it a single one for simplicity. 1-1 relationships are easier to understand
do you think we should have more than one?

Reply:
i made it a list